### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by delboy1978uk**

### CODE REVIEW

**Comment by tienvx** [comment]
Hi @delboy1978uk 

Thanks for your contributing.

Currently I'm focusing on developing Pact-PHP version 9.0, so I can't review this right now. I'm sorry for that.

I will comeback to this project sometime in the future. Until that, this is a way you can use your code without waiting for my reviewing:

```json
# composer.json
    "require-dev": {
        "tienvx/codeception-pact-php-extension": "dev-master as 1.0.4"
    },
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/delboy1978uk/codeception-pact-php-extension.git"
        }
    ],
```

File: src/Extension/Pact.php [hunk 1]
```diff
@@ -72,18 +72,28 @@ public function initSuite(SuiteEvent $e)
         }
     }
 
+    private function shouldUseLocalServer(): bool
+    {
+        return \in_array($this->mockServerConfig->getHost(), ['localhost', '127.0.0.1']);
+    }
+
```
Comment (delboy1978uk) [inline src/Extension/Pact.php:79]: this  method `shouldUseLocalServer` is based around this comment https://github.com/pact-foundation/pact-php/issues/152#issuecomment-645318929

**Comment by delboy1978uk** [comment]
Thanks  @tienvx, that is exactly what I'm doing!  I have however added more code that the ticket description does not warrant for,  based upon issues I received whilst using the library, i've commented where necessary and look forward to a quick review

**Comment by delboy1978uk** [comment]
Hi @tienvx are you able to have a look at this PR yet?

**Comment by tienvx** [comment]
Thanks @delboy1978uk

**Pull request merged**